rosbag_manager/rosbag_manager/config/bags_config.yaml

```diff
@@ -14,3 +14,5 @@ split: true
 split_size: 1024
 compressed: true
 regex: true
+max_hdd_percent_recording: 90
+max_hdd_percent: 85
```

Comment:
for others, the name for the variables are:

max_disk_usage
max_disk_usage_recording